### DISCUSSION TITLE
Is it possible to add a freebsd13 binaryTarget?

### DISCUSSION THREAD

**Discussion by dcschmid** [prisma/prisma#11879] [Q&A] [answered]
Possible binaryTargets: darwin, darwin-arm64, debian-openssl-1.0.x, debian-openssl-1.1.x, rhel-openssl-1.0.x, rhel-openssl-1.1.x, linux-arm64-openssl-1.1.x, linux-arm64-openssl-1.0.x, linux-arm-openssl-1.1.x, linux-arm-openssl-1.0.x, linux-musl, linux-nixos, windows, freebsd11, freebsd12, openbsd, netbsd, arm, native

**Comment by nurul3101** [comment, marked as answer]
Hey @dcschmid :wave:

Unfortunately we don't support freebsd13 as of now.
We already have an open issue for providing freebsd support.
- #3091

Can you please leave a 👍🏻 to the issue so that our engineering and product team could prioritise it?
Please let me know if there are any specific issues in using freebsd12 or freebsd11 version?
Also you could refer to this [reference](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#binarytargets-options) to know about supported binaryTargets

**Reply by dcschmid** [reply]
Hi @nurul3101,,

thanks for your answer, i do the prism Rust build what is written in the issue and set my eng vars. But when i run it, it do the migration but after that it shows me that error. Would be great to geht a FreeBSD 13 support.

**Reply by nurul3101** [reply]
@dcschmid Can you please create a new [GitHub Issue](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with the exact error messages that you are getting and steps for reproducing it?
So that our engineering team could have a look at it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
